vendor/brave-ios/BATBraveRewards.mm

```diff
@@ -91,7 +91,7 @@ - (instancetype)initWithConfiguration:(BATBraveRewardsConfiguration *)configurat
                              adsClass:(nullable Class)adsClass
 {
   if ((self = [super init])) {
-    if (!base::MessageLoopCurrent::Get()) {
+    if (!base::CurrentThread::Get()) {
```

Comment:
cc @Brandon-T Sync will have these changes in c86 instead as you moved this

Reply:
@kylehickinson I'll take care of it, it will cause a build error anyway